src/languageservice/parser/jsonParser07.ts

```diff
@@ -23,6 +24,8 @@ import { URI } from 'vscode-uri';
 import { DiagnosticSeverity, Range } from 'vscode-languageserver-types';
 import { TextDocument } from 'vscode-languageserver-textdocument';
 import { Diagnostic } from 'vscode-languageserver';
+import { MissingRequiredPropWarning, TypeMismatchWarning, ConstWarning } from '../../../test/utils/errorMessages';
```

Comment:
If you are need types form tests code, then it would be better to extract them in to `src` folder

Reply:
I removed this import, didn't notice it before.
Values are defined also in jsonParser07 file without reference to test file...